src/main/java/org/entando/entando/web/user/model/UserRequest.java

```diff
@@ -32,6 +32,8 @@
     //@Size(min = 6, message = "user.password.size")
     private String password;
 
+    private String passwordConfirm;
```

Comment:
better `passwordConfirmation`

Reply:
I am matching the parameter name that I receive from the UI. This is also specified in the acceptance criteria in the ticket.

Reply:
mmm ok